app/controllers/users_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @post_most_recent = Post.where(author_id: params[:id]).limit(3).order(created_at: :desc)
```

Comment:
- [ ] As I mentioned earlier, the same applies to this method. Kindly remove it and  instead make use of the method that you created initially in the previous milestone in the `User Model` since that was precisely the reason why you were asked to create those methods in the earlier milestones :+1: 
- [ ] I have pasted the method below :+1: 

```ruby
  def recent_post
    Post.where(author_id: id).limit(3).order(created_at: :desc)
  end
```